config/types/update.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2016 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package types
+
+import (
+	"errors"
+	"fmt"
+	"net/url"
+	"strings"
+
+	ignTypes "github.com/coreos/ignition/config/v2_0/types"
+	"github.com/coreos/ignition/config/validate/report"
+	"github.com/vincent-petithory/dataurl"
+)
+
+var (
+	ErrUnknownChannel = errors.New("unknown update channel")
+)
+
+type Update struct {
+	Channel UpdateChannel `yaml:"channel"`
+	Server  UpdateServer  `yaml:"server"`
+}
+
+type UpdateChannel string
+type UpdateServer string
+
+func (c UpdateChannel) Validate() report.Report {
+	switch strings.ToLower(string(c)) {
+	case "stable", "beta", "alpha":
```

Comment:
They might be using a custom instance of CoreUpdate with different group names. We could warn if they haven't overridden the server and don't use one of the standard groups.